source/extensions/omni.isaac.lab/omni/isaac/lab/utils/math.py

```diff
@@ -8,11 +8,15 @@
 # needed to import for allowing type-hinting: torch.Tensor | np.ndarray
 from __future__ import annotations
 
+import math
 import numpy as np
 import torch
 import torch.nn.functional
 from typing import Literal
 
+import omni.isaac.core.utils.stage as stage_utils
+from pxr import UsdGeom
```

Comment:
I think it is better to keep this file free from any Kit related functions. Reason being: utils were designed to general things that don't require anything from Omniverse.

You can take an input argument for the up-axis inside the respective function?